service/worker/scanner/shardscanner/scanner.go

```diff
@@ -162,6 +162,9 @@ func (s *ShardScanner) Scan(ctx context.Context) ScanReport {
 				Execution: execution,
 				Result:    checkResult,
 			}); err != nil {
+				// This looks like one of the contributors of the high number of skipped_scans in the system.
+				// Upon closer investigation we discovered that there is no retry mechanism in the Blobstore write function, which might
+				// result in unnecessary skip of some scans. Especially, the ones that might have happened due to network blip.
```

Comment:
probably worth removing this, as it's (at least partially) no longer true after this merges.

useful as a commit message / review context, but not really for long-term purposes